### PR TITLE
[i18n] LB Help

### PR DESCRIPTION
**Pull request by delphiactual**
Localized LB-Help
 - [x] English
 - [x] German
 - [x] Italian
 - [x] Spanish
 - [ ] French
 - [ ] Portuguese
 - [ ] Japanese

### CODE REVIEW

**Comment by delphiactual** [comment]
figured out the styling, added in the language templates.
Feedback requested.

screenshot 2016-11-13 13 01 13

**Comment by delphiactual** [comment]
@brunnopleffken I saw you css magic you did on #1192 is there any way you can help me out with the styling on this?

**Comment by brunnopleffken** [comment]
@delphiactual Sure I can!! I'll fork your repository and see what I can do...

File: app/views/best.html [hunk 1]
```diff
@@ -98,29 +98,17 @@ <h1 translate="LB"></h1>
...
-          </ul>
-        </li>
-        <li>Try to find armor that has ammo increasing perks for weapon types that you use.</li>
-        <li>Change the Intellect, Discipline, or Strength nodes in game to what is displayed to create each loadout.</li>
-        <li>As an items defense level increases, the stats on that item (int/dis/str) also increase.</li>
-      </ul></span>
+      <span><div ng-show="normal.showHelp">
+        <iframe ng-show="normal.lang === 'en'" src="/views/en/best-help.html" width="80%" height="280em" scrolling="no" style="border:none" name="showHelp" id="showHelp"></iframe>
+        <!--
+          <iframe ng-show="normal.lang === 'de'" src="/views/de/best-help.html" width="80%" height="280em" scrolling="no" style="border:none" name="showHelp" id="showHelp"></iframe>
```
Comment (bhollis) [inline app/views/best.html]: Check out `ng-switch`

Comment (bhollis) [inline app/views/best.html]: Actually, why not use `ng-src` and just interpolate the language?

Comment (bhollis) [inline app/views/best.html]: Also, the iframe is a cool idea, but why not just load the right file directly into the template?

Comment (delphiactual) [inline app/views/best.html]: @bhollis I did not realize that was possible

**Comment by delphiactual** [comment]
@brunnopleffken Using ngInclude removed the need to add styling, can you please look at the `pt-br best-help.html` and translate it?

**Comment by delphiactual** [comment]
@bhollis, @SunburnedGoose, @kyleshay AFAIK this is ready unless you have any other feedback. #1211 was already merged, so rebased on that.

**Pull request merged**